client/src/app/keymanagement/keylist-details/keylist-details.component.ts

```diff
@@ -5,6 +5,7 @@ import {Subject} from 'rxjs';
 import {takeUntil} from 'rxjs/operators';
 import {KeyItemEto} from '../common/to/KeyItemEto';
 import {KeyListCto} from '../common/to/KeyListCto';
+import {KeymanagementRestService} from '../keymanagement.rest.service';
```

Comment:
Is this crappy naming still state-of-the-art? I thought that after some time finally even the angular makers gave in and accepted TS conventions.

Reply:
It looks so. the .service part is most likely added by Angular CLI during generation. Unsure if it should be like this: keymanagement.rest.service.ts - to me it should be rather keymanagement-rest.service.ts